.bash_prompt

```diff
@@ -164,7 +164,15 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  #remote_branch="origin/$branch"
+  #Compute the remote tracking branch, if no remote branch then echo 0
+  branchnamefull=$(git symbolic-ref -q HEAD);
+  remote_branch=$(git for-each-ref --format='%(upstream:short)' $branchnamefull)
```

Comment:
Instead of using two commands, we should be able to use 1 command:

```bash
# https://git-scm.com/docs/git-rev-parse#git-rev-parse-emltbranchnamegtupstreamemegemmasterupstreamememuem
remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}")"
```

This was inspired by http://stackoverflow.com/a/9753364 by taken further by looking into the corresponding `man` page